<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HWTrainUI</class>
 <widget class="QMainWindow" name="HWTrainUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Kp">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>100</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kp</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="KiText">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>120</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>01000</string>
    </property>
   </widget>
   <widget class="QLabel" name="Engineer">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt; font-weight:600;&quot;&gt;Engineer UI&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="KpText">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>01000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enterButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QLabel" name="Ki">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>100</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ki</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ports">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="commandedSpeed">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>417</x>
      <y>220</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>20</width>
      <height>231</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Commanded Speed</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
